zScanner/SupportFiles/Extensions/UIViewController.swift

```diff
@@ -12,4 +12,14 @@ extension UIViewController {
     var safeArea: UILayoutGuide {
         return view.safeAreaLayoutGuide
     }
+    
+    func hideKeyboardWhenTappedAround() {
```

Comment:
The name is not clear. It's not describing what are you doing. 
Don't put condition into the function name. This function do: setup gesture recognizer for tap. What's gonna happen with keyboard.
So something like `setupKeyboardHandling` or `setupTapGesture` or something that.